eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/io/AccessionedVcfFactory.java

```diff
@@ -0,0 +1,97 @@
+package uk.ac.ebi.eva.accession.core.io;
```

Comment:
I would suggest to either put `AccessionedVcfFactory` and `AccessionedVcfLineMapper` in package `uk.ac.ebi.eva.accession.core.batch.io` or to create a different package with a meaningful name. Just to avoid having 2 `io` packages

Reply:
moved to the already existing uk.ac.ebi.eva.accession.core.batch.io package